src/main/java/collectors/RegionalHouseholdStats.java

```diff
@@ -129,7 +158,45 @@ public void record() {
                     homelessAnnualisedTotalIncome += h.getMonthlyPreTaxIncome();
                 }
             }
+            // check if the househld is a commuter travelling from one region to another
+            House home=h.getHome();
+            Region region=h.getRegion();
+            if(region!=null && home !=null){
+            	if(home.getRegion().getRegionId()!=region.getRegionId()){
+            		this.nCommuter++;
+            	}
+            }
+            // sum the monthly travl cost of each household who need to commute
+            double monthlyTravelCost=h.getMonthlyTravelCost(transport);
+            if(monthlyTravelCost>0){
+            	sumMonthlyTravelCost+=monthlyTravelCost;
+            	cntVaildMonthlyTravelCost++;
```

Comment:
Yes, it currently computes the  average monthly travel fee and travel cost only over those who commute, rather than over all agents. However, both indicators are important, as you mentioned.
For the current indicator, it will also change when households purchase houses in different regions due to the different commuting cost and time.

Reply:
Ok, that's fine, just make sure you make it very clear in the variable names and in the comments which one you're referring to.

Reply:
Sure.